src/iptvsimple/Settings.cpp

```diff
@@ -0,0 +1,116 @@
+#include "Settings.h"
```

Comment:
copyright header missing

Reply:
hmmm?

Reply:
Use what er use everywhere else in the addon.